public/contribute.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "portal-alpha",
+  "description": "File Sharing Experiment",
+  "repository": {
+    "url": "https://github.com/mozilla/something-awesome/",
+    "license": "MPL-2.0"
+  },
+    "participate": {
+      "home": "https://github.com/mozilla/something-awesome/blob/master/README.md",
+      "docs": "https://github.com/mozilla/something-awesome/blob/master/README.md"
+    },
+    "bugs": {
+      "list": "https://github.com/mozilla/something-awesome/issues",
+      "report": "https://github.com/mozilla/something-awesome/issues/new"
+    },
+    "urls": {
+      "prod": "https://fileshare.stage.mozaws.net/",
+      "stage": "https://fileshare.stage.mozaws.net/",
+      "dev": "https://fileshare.dev.mozaws.net/"
```

Comment:
These URLs (and any references to "something-awesome") will need to be updated per #7 and #166. Slapping "WIP:" on this PR so I can modify as #7 checklist completes.